src/uadk_digest.c

```diff
@@ -30,6 +30,11 @@
 #include "uadk_async.h"
```

Comment:
Do we need to use a separate .c file to store the provider implementation of each algorithm?